frameworks/elastic/tests/test_searchguard.py

```diff
@@ -0,0 +1,152 @@
+from typing import Any, Dict, Iterator
```

Comment:
Do these tests depend on version and type of DCOS (open enterprise)? 
Nightly Build failures can be occur and u can use marks like here https://github.com/mesosphere/dcos-elastic-service/blob/master/frameworks/elastic/tests/test_kibana.py#L177

Reply:
thanks, done!